src/assets/i18n/en_US.json

```diff
@@ -1223,7 +1223,7 @@
   "shopping_cart.ministatus.view_cart.link": "View cart",
   "shopping_cart.oci.transfer_basket.button": "Transfer cart",
   "shopping_cart.paging.items.label": "{{0, plural, one{# line item} other{# line items}}}",
-  "shopping_cart.payment.canceled.text": "Your payment was canceled. You can continue with the checkout or use the express checkout.",
+  "shopping_cart.payment.canceled.text": "Your payment was canceled. Please select another payment method. You can continue with the checkout or use the express checkout.",
```

Comment:
I added this sentence because it was there in DE (but missing in EN and FR). Not sure if it was overlooked in DE and should actually have been deleted there.

Reply:
I have no idea. The ones that might have an idea are not available right now, so we take your change as improvement.